weaviate/collections/collection/sync.py

```diff
@@ -140,9 +146,14 @@ def __init__(
         """This namespace includes all the CRUD methods available to you when modifying the tenants of a multi-tenancy-enabled collection in Weaviate."""
 
     def __len__(self) -> int:
-        total = self.aggregate.over_all(total_count=True).total_count
-        assert total is not None
-        return total
+        if self._connection._weaviate_version.is_lower_than(1, 28, 4):
+            total = self.aggregate.over_all(total_count=True).total_count
+            assert total is not None
+            return total
+        else:
+            return _EventLoopSingleton.get_instance().run_until_complete(
+                self.__aggregate_grpc.objects_count
+            )
```

Comment:
same here. I'D expect that `self.aggregate.over_all` switches to GRPC when it is available